static-site/README.md

```diff
@@ -4,10 +4,11 @@
 
 <br/>
 
-
+# Nextstrain static site
 This directory contains the documentation files and static site generator for nextstrain.
+For general `nextstrain.org` documentation including the nextstrain server (charon) and auspice customizations, see [nextstrain.org/README.md]((./static-site/README.md)).
```

Comment:
Given the recent confusion over what "charon" is, can we reword this? (Charon's not the "nextstrain server" -- it's just the word which refers to the auspice-server API)